README.md

```diff
@@ -1,2 +1,70 @@
-# Hello-World
-setup of my first microvesre project
+
+Microverse
+
+
+microverse project setup
+
+this is the project which includes all necessary steps a micronaut must have at beginning.
+
+
+
+
+
+Built With
+Major languages:html and css
+
+
+
+
+Technologies used:git,github,linters.
+Live Demo
+
+Live Demo Link
+
+Getting Started
+
+
+in order to run project locally you first have to clone it to your computer after run it in web browser by opening  index.html 
+
+
+
+Prerequisites
+
+
+Setup:no setup present.
+Install:you need to clone the project files to your laptop and after you run them locally in your laptop
+
+Run tests:no tests required.
+
+Authors
+
+👤 Author
+GitHub:https://github.com/mugishasam123/
+
+
+Twitter: https://twitter.com/mugishasamuel42/
+
+
+LinkedIn: https://www.linkedin.com/in/mugisha-samuel-55a905208/
+
+
+
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+I created all codes myself
+
+
+Inspiration:from microverse
+etc
+
+📝 License
+
+This project is open source project.
```

Comment:
- [ ] To get your project approved you can make your `README.md` file looks professional. You can use the [Microverse `README.md` Template](https://github.com/microverseinc/readme-template). 
- [ ] I can see that you are having trouble setting your headers inside your `README.md` file, you can go over to the [readme cheat sheet](https://www.markdownguide.org/cheat-sheet/). Make use of that on your headers and if there is something you don't need, for example: "live demo link", just remove them for your code.

***Remember the template is just for you to reference, feel free to make changes according to your project***